src/main/resources/graphics/scenery/volumes/ComputeHistogram.comp

```diff
@@ -0,0 +1,47 @@
+#version 450
+#extension GL_EXT_debug_printf : enable
+
+
+layout (local_size_x = 16, local_size_y = 16) in;
+layout (set = 0, binding = 0, r16f) uniform image3D Volume16Bit;
+layout (set = 1, binding = 0, r8) uniform image3D Volume8Bit;
+layout (set = 2, binding = 0, r32i) uniform iimage3D Histogram;
+
+layout(set = 3, binding = 0) uniform ShaderProperties {
+    bool volumeIs8Bit;
+    int numVoxels;
+    float maxDisplayVal;
+    float minDisplayVal;
+    int numBins;
+};
+
+ivec2 debug_pixel = ivec2(512,512);
+
+
+void main() {
+
+    if(gl_GlobalInvocationID.xy == debug_pixel){
+        debugPrintfEXT("Hello World!!!!!!!!!!!!!!!!!!!!!!!!!!");
```

Comment:
This is missing at least 10 exclamation marks ⚠️

No, but seriously, remove this :-D

Reply:
115eb7066dd07530e6125e03f7002dab377d1f16 adds necessary exclamation marks

No, removes debug code :)